packages/typescript-client/test/cache.test.ts

```diff
@@ -105,6 +107,79 @@ describe(`HTTP Proxy Cache`, { timeout: 30000 }, () => {
     expect(getCacheStatus(cachedRes)).toBe(CacheStatus.HIT)
   })
 
+  it.only(`should collapse requests in live mode`, async ({
```

Comment:
```suggestion
  it(`should collapse requests in live mode`, async ({
```

Is this test still useful?

Reply:
I think so, good to have as a safety check like the other cache ones - I mostly added it cause I realised we didn't have that anywhere